ssl/ssl_lib.c

```diff
@@ -5862,8 +5864,11 @@ const EVP_CIPHER *ssl_evp_cipher_fetch(OPENSSL_CTX *libctx,
     }
 #endif
 
-    /* Otherwise we do an explicit fetch */
-    return EVP_CIPHER_fetch(libctx, OBJ_nid2sn(nid), properties);
+    /* Otherwise we do an explicit fetch. This may fail and that could be ok */
```

Comment:
Uhm, why is that ok?

Reply:
Because in libssl we sometimes try to get a cipher which we know is not available on all platforms.

Reply:
Ah, all right